spec/sanity/events.js

```diff
@@ -13,8 +13,10 @@ describe('logs & events page', function() {
         browser.sleep(500);
     }
 
-    beforeEach(function(){
-        events.route();
+    beforeEach(function(done){
+        components.ui.LoginPage.goTo().login('admin', 'admin');
+        components.ui.layout.goToLogsEvents();
```

Comment:
Should we route to pages with clicking the sidebar rather then routing to a url?